goldilocks/src/extension.rs

```diff
@@ -27,7 +26,7 @@ impl HasTwoAdicBinomialExtension<2> for Goldilocks {
         if bits == 33 {
             [Self::ZERO, Self::new(15659105665374529263)]
         } else {
-            [Self::TWO_ADIC_GENERATORS[bits], Self::ZERO]
+            [TWO_ADIC_GENERATORS[bits], Self::ZERO]
```

Comment:
I think you can just call the function from the base field here (after adding `use p3_field::TwoAdicField`):
```suggestion
            [<Self as TwoAdicField>::two_adic_generator(bits), Self::ZERO]
```
Then `TWO_ADIC_GENERATORS` doesn't need to be `pub(crate)` below.